gradio/cli/commands/components/publish.py

```diff
@@ -112,8 +115,9 @@ def _publish(
             help="HuggingFace token for uploading demo. Can be omitted if already logged in via huggingface cli."
         ),
     ] = None,
+    prefer_local: Annotated[bool, Option(help="Install")] = False,
```

Comment:
I think you forgot to finish the help string here

Reply:
ops